packages/core/src/App/Components/Layout/Header/__tests__/account-info-wrapper.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import AccountInfoWrapper from '../account-info-wrapper';
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Popover: jest.fn(({ children }) => <div>Popover: {children}</div>),
+}));
+
+describe('AccountInfoWrapper component', () => {
+    it('should render children when is_disabled and disabled_message are undefined', () => {
+        render(<AccountInfoWrapper>children</AccountInfoWrapper>);
+        expect(screen.queryByText(/children/i)).toBeInTheDocument();
```

Comment:
```suggestion
        expect(screen.getByText(/children/i)).toBeInTheDocument();
```

Reply:
@aizad-deriv please check the image from jest documentation

<img width="996" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/efbceda9-4605-45a5-a6a0-9e4f544c4f9f">

There is no difference between these 2 methods except one, `getBy` can throw the error if the element will not be founded. But in my test I'm checking `toBeInTheDocument` and `not.toBeInTheDocument` and for 2nd case I must use `queryBy` instead of `getBy`. In result if there is no difference and I need to use `queryBy` in one place I don't see the sense to use `getBy` in the same test case. Maybe you have a strong cause to use `getBy` instead of `queryBy`, please share with me because I don't see any difference.